api/src/routers/meals.js

```diff
@@ -0,0 +1,70 @@
+import express from "express";
+import knex from "../database_client.js";
+
+const router = express.Router();
+
+// router to get all the meals
+router.get("/", async (req, res, next) => {
+  try {
+    const meals = await knex.select("*").from("Meal");
```

Comment:
One suggestion is to not make implicit methods or values ​​explicit.


```suggestion
    const meals = await knex.from("Meal");
```